### DISCUSSION TITLE
Can someone explain to me how to redeem rewards on multiple accounts?

### DISCUSSION THREAD

**Discussion by Caumanuel** [charlesbel/Microsoft-Rewards-Farmer#379] [Q&A]
Can someone explain to me how to redeem rewards on multiple accounts? I have several phone numbers to be able to redeem the accounts but every time I try to redeem, after 24 hours I receive an email from Microsoft stating that there has been a problem with my order and they close my account. Thanks!

**Comment by VictimaVitae** [comment]
> Can someone explain to me how to redeem rewards on multiple accounts? I have several phone numbers to be able to redeem the accounts but every time I try to redeem, after 24 hours I receive an email from Microsoft stating that there has been a problem with my order and they close my account. Thanks!

**Only redeem once per day**, otherwise Microsoft, who are by no means the quickest horse in the race, will suspend your account and remove your points.
